src/configparser.js

```diff
@@ -36,7 +36,9 @@ const writeFileAsync = util.promisify(fs.writeFile);
 /**
  * @constructor
  */
-function ConfigParser() {
+function ConfigParser(options) {
+	options = options || {};
+	this._transform = options.transform || 'none';

```

Comment:
4 Space Indentation please, and we could just use default params, so `function ConfigParser(options = {}) {...}`